### ISSUE TITLE
Addition of Lyso-DAGs, DAG-OH and MAG-OH lipids to database.

### ISSUE THREAD

**Issue by j-e-hunter** [vanmooylipidomics/LOBSTAHS#19]
Expansion of the database and database generation components to include Lyso-forms of polar-diacylglyceride (Lyso-DAG), diacylglycerols (DAG-OH) and monoacylglycerols (MAG-OH).

**Comment by jamesrco** [comment]
This is a great idea! I did at some point add capacity for an "IP_MAG" class, but only incorporated one instance of it (lysophosphatidylcholine; LPC) into the component table and adduct hierarchy tables.

**Comment by j-e-hunter** [comment]
Done, see pull request:
https://github.com/vanmooylipidomics/LOBSTAHS/pull/20

**Comment by jamesrco** [comment]
Sorry, reopening this one. Good idea to leave issues open until they're fully resolved (i.e., incorporated into master or we reject them).

**Comment by j-e-hunter** [comment]
Right, will do!

**Comment by jamesrco** [comment]
I'm going to mark this as resolved. Accomplished via @j-e-hunter and @lee-t commits  8c6eecb28867ae54f6844c139c55518acb350c2d...0b0c25364b6d6fe1afd7f9f3b3667eecba5d68c8 and some additional work to fix merge issues by @jamesrco in  56e9192d5c119885b42bc1475b7c9f108081a8fd...17e5308e08d02d62c7928cfe51d5762093e354f2. Figured this was significant enough so I bumped version to v1.3.6 in 63e9133adba01f2654d8ebc176a1a7505acd1011.
